packages/eas-cli/src/commands/build/index.ts

```diff
@@ -104,6 +108,10 @@ export default class Build extends EasCommand {
     }
     const flags = this.sanitizeFlags(rawFlags);
 
+    const { actor } = await this.getContextAsync(Build, {
```

Comment:
I don't think we ever had a use-case where this value couldn't be global, I understand the reluctance to use the global state for anything, but if we would need to pass it everywhere it might be a bit messy

Reply:
In general global state for logged-in user makes a few things nearly impossible:
- switching users (switching logged-in user in a running program)
- mocking users (though jest is ok at this, injection is still a better IMO)

One other argument against global state is that we'd have to type it as `Actor | null` globally, while this context allows us to type it affirmatively.